htdocs/takepos/index.php

```diff
@@ -895,7 +899,7 @@ function WeighingScale(){
 	<div class="modal-body">
 		<button type="button" class="block" onclick="location.href='index.php?setterminal=1'"><?php print $langs->trans("Terminal"); ?> 1</button>
 		<?php
-		for ($i = 2; $i <= $conf->global->TAKEPOS_NUM_TERMINALS; $i++) {
+		for ($i = 2; $i <= getDolGlobalInt("TAKEPOS_NUM_TERMINALS"); $i++) {
```

Comment:
Adding a dynamic function inside the condition of a loop is forbidden.
Just do $iendofloop = getDolGlobalInt("TAKEPOS_NUM_TERMINALS") and use $iendofloop inside the for (...